server/routes/index.ts

```diff
@@ -28,6 +28,12 @@ export default function routes(services: Services): Router {
   get('/', homepageController.displayHomepage())
   post('/search', homepageController.search())
 
+  get('/dps-services-store', async (req, res, next) => {
```

Comment:
Would be better to have this identifiable as being an API route, so something like `/api/dps-services`, and for it to be handled by a new API controller rather than the `homepageController` especially as other services will likely be calling this endpoint. That way we can better handle API auth, etc and just keep it separate from the in-app stuff.
Happy for this to come in a later PR though if this is just the first step.